src/main/java/cz/codecamp/logger/Formatter.java

```diff
@@ -0,0 +1,45 @@
+package cz.codecamp.logger;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.List;
+
+
+public class Formatter implements FormatterInterface {
+
+    private StringBuilder builder;
+
+    @Override
+    public String format(LogLevelEnum level, String message) {
+        builder = new StringBuilder();
+        builder.append(getClassPath()).append(" ");
+        builder.append("[").append(level.name()).append("]").append(" ");
+        builder.append("[").append(df.format(new Date())).append("]").append(" ");
+        builder.append(message).append("\n");
+        return builder.toString();
+    }
+
+    private String getClassPath() {
```

Comment:
- Přes pole `StackTraceElement[]` vracené `Thread.currentThread().getStackTrace()` se dá jednoduše iterovat pomocí `for (StackTraceElement el : Thread.currentThread().getStackTrace())`, není imo potřeba řešit iterátor.
- `contains("cz.codecamp")` + `contains("formatter")` se mi moc nezdá...